app/src/main/java/com/sun/noteapp/ui/home/MainActivity.kt

```diff
@@ -103,7 +103,9 @@ class MainActivity : AppCompatActivity(),
     }
 
     override fun showNoteDetail(position: Int, note: Note) {
-        showToast(note.toString())
+        startActivity(TextNoteActivity.getIntent(this).apply {
```

Comment:
Xử lý việc truyền dữ liệu này trong hàm getIntent()
```suggestion
        startActivity(TextNoteActivity.getIntent(this, note));
```